source/raw_assets/Persona-Claudia.user.js

```diff
@@ -0,0 +1,86 @@
+// ==UserScript==
+// @name         Persona Claudia
+// @namespace    https://github.com/alphagov/accessibility-personas
+// @version      1.0.0
+// @license      MIT
+// @author       Crown Copyright (Government Digital Service)
+// @description  Claudia, a sight impaired screen magnifier user - magnifies the screen, changes colours to increase contrast, highlights the mouse pointer
+// @homepageURL  https://alphagov.github.io/accessibility-personas/
+// @include      *
+// @grant        GM_addStyle
+// @nocompat     Chrome
+// ==/UserScript==
+
+/**
+ * Invert colours
+ * @author Crown Copyright (Government Digital Service)
+ * @license MIT
+ */
+
+GM_addStyle('html, img, video, iframe { filter: invert(1) hue-rotate(180deg) !important; }');
+
+
+/**
+ * Zoom in 400% and widen the page
+ * @author Crown Copyright (Government Digital Service)
+ * @license MIT
+ */
+
+GM_addStyle('body { zoom: 4 !important; min-width: 200vh !important; }');
+
+
+/**
+ * Make focus more prominent
+ * @author Crown Copyright (Government Digital Service)
+ * @license MIT
+ */
+
+GM_addStyle('*:focus { outline: 5px solid #c00 !important; }');
+
+
+/**
+ * Highlight the mouse pointer
+ * @author Crown Copyright (Government Digital Service)
+ * @license MIT
+ */
+
+let css = `
```

Comment:
```suggestion
const css = `
```

Reply:
Thanks, I've fixed it in a rebased version (of now 240f793).